### PR TITLE
[improve][doc] Add broker level metrics statistics and expose to prometheus

### PR DESCRIPTION
**Pull request by yyj8**
Add doc for PIP https://github.com/apache/pulsar/pull/19047

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@congbobo184 Could you please review this PR from a technical perspective? Then writers will review it from a technical writing side. Thank you!

**Comment by Anonymitaet** [comment]
ping @congbobo184

**Review by congbobo184 — APPROVED** [review]

File: docs/reference-metrics.md [hunk 1]
```diff
@@ -148,6 +148,19 @@ All the broker metrics are labeled with the following labels:
...
 | pulsar_broker_throttled_connections_global_limit | Gauge | The number of throttled connections due to per-connection limit. |
+| pulsar_broker_topics_count | Gauge | The number of Pulsar topics in this broker. |
+| pulsar_broker_subscriptions_count | Gauge | The number of Pulsar subscriptions in this broker. |
+| pulsar_broker_producers_count | Gauge | The number of active producers connected to this broker. |
+| pulsar_broker_consumers_count | Gauge | The number of active consumers connected to this broker. |
+| pulsar_broker_rate_in | Gauge | The total message rate coming into this broker (message per second). |
+| pulsar_broker_rate_out | Gauge | The total message rate going out from this broker (message per second). |
+| pulsar_broker_throughput_in | Gauge | The total throughput coming into this broker (byte per second). |
+| pulsar_broker_throughput_out | Gauge | The total throughput going out from this broker (byte per second). |
+| pulsar_broker_storage_size | Gauge | The total storage size of all topics in this broker (bytes). |
```
Comment (Anonymitaet) [inline docs/reference-metrics.md:159]:
```suggestion
| pulsar_broker_storage_size | Gauge | The total storage size of all topics in this broker (in bytes). |
```
Keep consistent

Comment (yyj8) [reply docs/reference-metrics.md]: > Keep consistent

hi,  @Anonymitaet 

Modification completed

**Review by tisonkun — APPROVED** [review]

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**